modules/ROOT/pages/patterns/shortest-paths.adoc

```diff
@@ -538,3 +538,84 @@ RETURN [stop IN nodes(p) | stop.name] AS r1,
++----------------------------+----+-----------------------------------------------------------------------------------------+----------------+------+---------+----------------+------------------------+-----------+---------------+
+| +ProduceResults            |  0 | p                                                                                       |            383 |  154 |       0 |              0 |                    0/0 |     0.158 |               |
+| |                          +----+-----------------------------------------------------------------------------------------+----------------+------+---------+----------------+------------------------+-----------+               |
+| +Projection                |  1 | (a) ((anon_12)-[anon_14]-(anon_13)-[anon_11]-())* (b) AS p                              |            383 |  154 |       0 |                |                    0/0 |     2.785 |               |
+| |                          +----+-----------------------------------------------------------------------------------------+----------------+------+---------+----------------+------------------------+-----------+               |
+| +StatefulShortestPath(All) |  2 | SHORTEST 1 GROUPS (a) ((`anon_5`)-[`anon_6`]-(`anon_7`)-[`anon_8`]-(`anon_9`)){1, } (b) |            383 |  154 |     655 |          19711 |                    1/0 |    70.908 | In Pipeline 1 |
+| |                          |    |         expanding from: a                                                               |                |      |         |                |                        |           |               |
+| |                          |    |     inlined predicates: b:Station                                                       |                |      |         |                |                        |           |               |
+| |                          +----+-----------------------------------------------------------------------------------------+----------------+------+---------+----------------+------------------------+-----------+---------------+
+| +NodeByLabelScan           |  3 | a:Station                                                                               |             10 |    9 |      10 |            376 |                    1/0 |     1.106 | In Pipeline 0 |
++----------------------------+----+-----------------------------------------------------------------------------------------+----------------+------+---------+----------------+------------------------+-----------+---------------+
+----
+
+
+
+
+Rewriting the query using a `CALL subquery`
+[source,cypher]
+----
+MATCH
+  (a:Station),
+  (b:Station)
+CALL {
+  WITH a, b
+  MATCH
+    p = ALL SHORTEST (a)(()-[]-()-[]-()){1,}(b)
+  RETURN p
+}
+RETURN p
+----
+
+makes use of `StatefulShortestPath(Into)`, because the CALL subroutine is executed for each incoming row.  Each incoming row matches `a` to exactly one station node and `b` to exactly one station node.
```

Comment:
```suggestion
makes use of `StatefulShortestPath(Into)`, because the `CALL` subquery is executed for each incoming row.  Each incoming row matches `a` to exactly one station node and `b` to exactly one station node.
```